crates/rpc/rpc/src/debug.rs

```diff
@@ -87,7 +88,7 @@ where
     ) -> EthResult<Vec<TraceResult>> {
         if transactions.is_empty() {
             // nothing to trace
-            return Ok(Vec::new())
+            return Ok(Vec::new());
```

Comment:
This looks to still be broken 😅 

How are you guys formatting these days?

Reply:
I used a much older version of rustfmt that I had installed via Mason, but recently upgraded and has this issue as well. For me the answer is manually undo these 🤷‍♂️

https://github.com/rust-lang/rustfmt/issues/6182